### PR TITLE
Add changes to lists and names

### PR DESCRIPTION
**Pull request by myoung34**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -24,23 +12,22 @@ resource "aws_elasticache_cluster" "redis" {
   parameter_group_name = "default.redis2.8"
   port                 = "6379"
   subnet_group_name    = "${aws_elasticache_subnet_group.default.name}"
-  security_group_ids   = ["${aws_security_group.redis.id}"]
+  security_group_ids   = "${var.security_group_ids}"
```
Comment (hectcastro) [inline main.tf:15]: Can you please add this to the `README`?

File: main.tf [hunk 2]
```diff
@@ -53,10 +40,12 @@ resource "aws_cloudwatch_metric_alarm" "cpu" {
   threshold           = "75"
 
   dimensions {
-    CacheClusterId = "${aws_elasticache_cluster.redis.id}"
+    CacheClusterName = "${var.cache_name}"
   }
 
-  alarm_actions = ["${split(",", var.alarm_actions)}"]
+  # for some reason trying to pass a list here causes the error `  * aws_cloudwatch_metric_alarm.cpu: alarm_actions: should be a list` in terraform 0.7.5
```
Comment (hectcastro) [inline main.tf:46]: Is this happening because the variable isn't defined as type `list`?

File: outputs.tf [hunk 3]
```diff
@@ -1,7 +1,3 @@
-output "cache_security_group_id" {
```
Comment (hectcastro) [inline outputs.tf:1]: Can you please remove this from the `README`?

**Review by hectcastro — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute these changes, @myoung34.

File: main.tf (hunk 2 above)
Comment (myoung34) [reply main.tf:46]: I think it might be a bug. Even defining it as a list causes the apply to fail

File: main.tf (hunk 1 above)
Comment (myoung34) [reply main.tf:15]: done

File: outputs.tf (hunk 3 above)
Comment (myoung34) [reply outputs.tf:1]: done

**Comment by hectcastro** [comment]
Apologies for the slow turnaround time.

We recently had to build out a Redis ElastiCache cluster for a new project and encountered the [elasticache_replication_group](https://www.terraform.io/docs/providers/aws/r/elasticache_replication_group.html) Terraform resource. For Redis, this seems a lot more desirable than the more primitive `aws_elasticache_cluster` resource because it optionally helps wire up a hot-standby.

Moving forward, we want to change this module to make use of the `elasticache_replication_group` resource. I'd like to merge this PR, but not cut another release until the `elasticache_replication_group` change is made, and then cut a release that increments the major version number.

**Review by hectcastro — APPROVED** [review]

**Comment by myoung34** [comment]
No worries, I'm not dependent on it any longer, just wanted to submit some things I found. Thanks!

**Pull request merged**